user-interface/src/main/java/life/qbic/datamanager/views/projects/project/info/OfferList.java

```diff
@@ -36,7 +36,8 @@ public OfferList() {
     title.addClassName("title");
     Span header = new Span(title, upload);
     header.addClassName("header");
-    addClassName("offer-list");
+    addClassName("offer-list-component");
+    delegateList.addClassName("offer-list");
```

Comment:
how and why was the css changed here?

Reply:
Excellent spot! 👀 This css distinction introduced here allows us to set the Virtuallist within the offer-list-component so it adjusts it's height according to the list content. 
Otherwise vaadin defaults the height to 400px which is a lot of unnecessary whitespace if only one offer/qc was provided